src/dmd/dmangle.d

```diff
@@ -79,7 +79,7 @@ private immutable char[TMAX] mangleChar =
     Tfunction    : 'F', // D function
     Tsarray      : 'G',
     Taarray      : 'H',
-    Tident       : 'I',
```

Comment:
I am wondering why this change does not seem to break tests.
Where was Tident used? in templates?

Reply:
It isn't used (https://github.com/dlang/druntime/pull/2847#discussion_r341837820)